packages/website/docs/Internal/Grammar.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Grammar
+urlcolor: blue
+author:
+  - Quinn Dougherty
+---
+
+Formal grammar specification, reference material for parser implementation.
+
+_In all likelihood the reference will have to be debugged as we see what tests pass and don't pass during implementation_.
+
+## Lexical descriptions of constants and identifiers
+
+- `<number>` ::= `[-]? [0-9]+ (.[0-9]+)? | [-]? [0-9]+ (.[0-9]+)? [e] [-]? [0-9]+`
+- `<symbol>` ::= `[a-zA-Z]+ [a-zA-Z0-9]?`
+- `<baseDistribution>` ::= `(normal(<number>, <number>) | beta(<number>, <number>) | triangular(<number>, <number>, <number>) | cauchy(<number>, <number>) | lognormal(<number>, <number>)) | uniform(<number>, <number>) | <number> to <number> | exponential(<number>)`
```

Comment:
Is baseDistribution basically ``genericDistribution``?

Reply:
I meant that these are like the atomic distribution constructors, the distinctions between sampleset/pointset/symbolic really matters at the expression level (but the parser is agnostic about that)

Reply:
taken out.